src/main/resources/static/js/gw.history.js

```diff
@@ -1,66 +1,66 @@
 /**
- * 
+ *
  * author: Z.S.
  * date: Mar 12 2021
- * 
+ *
  */
 
 GW.history = {
 
     deleteAllJupyter: function(hostid, callback){
 
         if(confirm("WARNING: Are you sure to remove all the history? This is permanent and cannot be recovered.")){
-            
+
```

Comment:
you should separate these space changes into another PR. Don't pollute the real purpose of this PR with all these irrelevant changes.